tests/helper.py

```diff
@@ -41,6 +41,7 @@ def setup_greenthreads(patch_threads=False):
 
     try:
         import eventlet
+        eventlet.hubs.get_hub()
```

Comment:
Why not add a comment why this is needed, linking to the eventlet patch adding this?